Pearson-Advance/openedx-themes
# PAE-157 - Styling the header of the Regis theme.

## Description:

This PR changes the Regis theme header to comply with the Regis mockups.

## Screenshots:

![image](https://user-images.githubusercontent.com/17520199/179172390-f9f25ac1-d238-48c0-922c-def4e20f40e4.png)
![image](https://user-images.githubusercontent.com/17520199/179172437-caa40060-841a-445f-b6d2-816c8a7fe0f6.png)
![image](https://user-images.githubusercontent.com/17520199/179327118-e59e3b28-2d5e-46a0-92ee-9a1ab8d30117.png)

## Testing:

- Check out this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets, `paver update_assets --themes=pearson-regis-theme`
- Assign 'pearson-regis-theme' to a new site.

## Reviewers:
